AnisimovAN/Lab1/Lab1/alfa.cpp

```diff
@@ -0,0 +1,42 @@
+#include <iostream> 
+#include <conio.h> 
+using namespace std;
+int main()
+{
+	float left, right, result;
+	char sign;
+	cout <<"Insert first argument" << endl;
+	cin >> left;
+	cout <<"Insert sign" << endl;
+	cin >> sign;
+	cout <<"Insert second argument" << endl;
+	cin >> right;
+
+	switch (sign)
+	{
+	case '+': result = left + right;
+		break;
+
+	case '-': result = left - right;
+		break;
+
+	case '*': result = left * right;
+		break;
+
+	case '/': result = left / right;
+		if (right == 0) {
+			cout << "Waring: it's not available to divine by 0";
+			_getch();
+			return 0;

```

Comment:
Do not use return statement here
Normally you don't wont calculator you use to shut down every time you make a mistake